drivers/input/keyboard/miyoo_kbd.c

```diff
@@ -561,6 +561,22 @@ static void scan_handler(unsigned long unused)
       val|= MY_R2;
       hotkey_actioned = true;
     }
+    if((val & MY_R) && (val & MY_SELECT)) {
```

Comment:
the default bindings are placed here https://github.com/MiyooCFW/kernel/blob/master/drivers/input/keyboard/miyoo_kbd.c#L661-L813

So for V90 this is just a duplicate (with that you just changed kill to shutdown R+SELECT combo).